app/models/api/transliterate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+// Check that the class exists before trying to use it
+if (! class_exists('Transliterator')) {
+    $request->error = 'Class Transliterator not present';
+    $json = $request->invalidAPICall();
```

Comment:
That method returns a 400 HTTP code error, which is a user on the user's end. But here we have to deal with a server-side error, I suggest returning the 501 error code. That means updating this method to have a default value of 400 and deal with the 501 error (and update tests).
ex:
```php
$json = $request->invalidAPICall(501);
......
public function invalidAPICall($error = 400)
    {
        http_response_code($error);

        return ['error' => $this->error];
    }
```